### ISSUE TITLE
Learning spells 

### ISSUE THREAD

**Issue by stormhalter** [jkachhad/Stormhalter#1077]
So, interestingly..

If you have multiple spells at you purchase them in order starting from 1 and up, you'll have a bad time. If you start from top, then work down, you'll be fine. Here's why:

1 - Spell 1
2 - Spell 2
3 - Spell 3

os, teach 1

you would think, this would happen:

2 - spell 2
3 - spell 3

but nope, this happens:

1 - spell 2
2 - spell 3

so you have to keep doing this:

os, teach 1
os, teach 1

kind of annoying when you just memorized what 1-2-3 really were in the beginning, you know? what if you didnt want a spell, it could throw things off.

Anyway, thanks for listening, great job Lux!! [Report by NotTaelamar]

---
[View on Discord](https://discord.com/channels/1056432310632841276/1530616504721215598/1530616504721215598)

---
[View on Discord](https://discord.com/channels/577503035526348810/1532154357401780377/1532154357401780377)

**Comment by Tawnos-LOK** [comment]
I agree that it is pretty jarring.  I don't like that mitigation is just hitting up + enter over and over on "x, teach 1"
